File: src/main/java/com/personal/web/entity/Car.java
package com.personal.web.entity;

import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@Entity
@NoArgsConstructor
public class Car {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long carSeq;
    @Column(length = 200)
    private String img;
    @Column(length = 100)
    private String name;
    @Column(length = 50)
    private String year;
    @Column(length = 50)
    private String km;
    @Column(length = 50)
    private String fuel;
    @Column(length = 50)
    private String price;
    @Builder
    public Car(String img,String name,String year,String km,String fuel,String price) {
        this.img = img;
        this.name = name;
        this.year = year;
        this.km = km;
        this.fuel = fuel;
        this.price = price;
    }
}

File: src/frontend/src/store/car.js
import axios from 'axios'
const state = {
    context : "http://localhost:5000",
    list : []
}
const actions = {
    async list({commit}){
        axios.get(`${state.context}/car/list`)
            .then(({data})=>{
                commit("LIST",data)
            })
            .catch((e)=>{
                console.log(e)
            })
    }
}
const mutations = {
    LIST(state,data){
        state.list = data
        data.forEach(item => {
            state.list.push({
                carSeq : item.carSeq,
                img : item.img,
                name : item.name,
                year : item.year,
                km : item.km,
                fuel : item.fuel,
                price : item.price
            })
        })
    }
}
export default {
    name:"car",
    namespaced : true,
    state,
    actions,
    mutations
}